src/cli.rs

```diff
@@ -20,34 +20,32 @@ use tokio_core::reactor::Core;
 
 /// Inner main function which can return an error.
 pub fn main() -> Result<()> {
-    let app = App::new(env!("CARGO_PKG_NAME"))
-        .setting(AppSettings::AllowExternalSubcommands)
+    let app = Command::new(env!("CARGO_PKG_NAME"))
+        // .setting(Command::is_allow_external_subcommands_set)
```

Comment:
From [`clap`'s doc](https://docs.rs/clap/latest/clap/struct.App.html#method.allow_external_subcommands) I assume we want `.allow_external_subcommands(true)` here. Or do we no longer need this setting?

Reply:
Yes, I was unsure about this and forgot to get back to it.